.github/workflows/acceptance.yml

```diff
@@ -106,9 +106,9 @@ jobs:
         uses: actions/cache@v2
         with:
           path: ${{ steps.pip.outputs.pip-cache-dir }}
-          key: ${{ runner.os }}-pip-py${{ steps.python-version.outputs.python-version }}-${{ hashFiles('**/requirements-*.txt') }}
+          key: ${{ runner.os }}-pip-py${{ steps.python-version.outputs.python-version }}-${{ steps.pip.outputs.pip-version }}-${{ hashFiles('**/requirements-*.txt') }}
           restore-keys: |
-            ${{ runner.os }}-pip-
+            ${{ runner.os }}-pip-py${{ steps.python-version.outputs.python-version }}-${{ steps.pip.outputs.pip-version }}-
```

Comment:
After various iterations on this PR, I realized that the only workflow I care about to include the pip version is the dev env workflow.

Please let me know if you want me to remove this change for all other workflows or if you would like me to keep it.

This change would simply mean that our Python cache will invalidate few more times per year and that might simply be fine:
<img width="629" alt="image" src="https://user-images.githubusercontent.com/44410/107825605-60475c00-6d51-11eb-87a7-49873612556f.png">

Reply:
I think this is fine to keep